yarn.lock

```diff
@@ -79,6 +71,8 @@ ajv-keywords@^3.1.0:
   resolved "https://registry.yarnpkg.com/ajv-keywords/-/ajv-keywords-3.1.0.tgz#ac2b27939c543e95d2c06e7f7f5c27be4aa543be"
   integrity sha1-rCsnk5xUPpXSwG5/f1wnvkqlQ74=
 
+<<<<<<< HEAD
```

Comment:
seems like something went wrong here, did you try to merge?
Was there supposed to be any changes in the yarn.lock file? if not and you don't know how to fix it, let me know and I will help you out.